### PR TITLE
3.1.4 from sdist

### PR DESCRIPTION
**Pull request by tomjakubowski**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -32,57 +22,28 @@ requirements:
...
-  #       - python
-  #     run:
-  #       - python
-  #       - {{ pin_subpackage("perspective", exact=True) }}
-
 about:
   home: http://github.com/finos/perspective
   license: Apache-2.0
   license_family: Apache
   license_file:
```
Comment (timkpaine) [inline recipe/meta.yaml:44]: lets start to get better about including our statically linked licenses:
- https://github.com/conda-forge/polars-feedstock/blob/993ca7886232c75d5ab67715644b490bd04ae84f/recipe/meta.yaml#L39
- https://github.com/conda-forge/polars-feedstock/blob/993ca7886232c75d5ab67715644b490bd04ae84f/recipe/build.sh#L72
- https://github.com/conda-forge/polars-feedstock/blob/993ca7886232c75d5ab67715644b490bd04ae84f/recipe/meta.yaml#L66

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (texodus) [reply recipe/meta.yaml:44]: This doesn't seem relevant to this PR, how about you open an issue on the feedstock repo instead please

Comment (timkpaine) [reply recipe/meta.yaml:44]: This needs to be done for rust projects so I would consider it relevant

Comment (timkpaine) [reply recipe/meta.yaml:44]: https://github.com/conda-forge/perspective-feedstock/issues/182

**Pull request closed without merging**